### PR TITLE
Add logic to setup admission webhooks in local test environment

### LINKED ISSUE #20: Missing go module sigs.k8s.io/kind
### Description

The required go module `sigs.k8s.io/kind` is missing:

```
no required module provides package sigs.k8s.io/kind; to add it:
	go get sigs.k8s.io/kind
make[1]: *** [/Users/gerbermichi/dev/control-api/local-env/../local-env/kind-kubeconfig-v1.22.1] Error 1
```

### Additional Context

_No response_

### Logs

```shell
no required module provides package sigs.k8s.io/kind; to add it:
	go get sigs.k8s.io/kind
make[1]: *** [/Users/gerbermichi/dev/control-api/local-env/../local-env/kind-kubeconfig-v1.22.1] Error 1
```

### Expected Behavior

I should not have to install the module manually.

### Steps To Reproduce

1. `make local-env-setup`
2. Accept default URL
3. Enter name `mig`
4. Import realm into keycloak
5. Create admin user in keacloak
6. Press the enter key in the terminal

### Version used

v1.22.1

### Version of OpenShift

default

### PR DESCRIPTION
**Pull request by simu**
## Summary

* Add `sigs.k8s.io/kind` to `tools.go` (Fixes #20)
* Add logic to setup admission webhooks on local env

## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: local-env/setup-kind.sh [hunk 1]
```diff
@@ -88,8 +90,40 @@ kubectl config set-context --current --user=oidc-user
...
+  labels:
+    app: webhook-service
+  name: webhook-service
+  namespace: control-api
+spec:
+  externalName: ${kind_network_gateway_ip}
+  selector:
+    app: webhook-service
+  type: ExternalName
+EOF
```
Comment (glrf) [inline local-env/setup-kind.sh:109]: This assumes that the control API runs on the host. However currently we deploy the control API to the created kind cluster. I don't think we want to change that do we?

**Review by glrf — CHANGES_REQUESTED** [review]

File: local-env/setup-kind.sh (hunk 1 above)
Comment (simu) [reply local-env/setup-kind.sh:109]: I didn't fully realize that. I'll update the PR.

File: local-env/setup-kind.sh [hunk 2]
```diff
@@ -88,8 +90,54 @@ kubectl config set-context --current --user=oidc-user
...
+  -p '{
+    "webhooks": [
+      {
+        "name": "validate-users.appuio.io",
+        "clientConfig": {
+          "caBundle": "'"$(base64 -w0 "${script_dir}"/webhook-certs/tls.crt)"'"
+        }
+      }
+    ]
+  }'
```
Comment (simu) [inline local-env/setup-kind.sh:137]: It's possible that we could apply these patches with Kustomize, but I don't know how I would have to do that.

Comment (glrf) [reply local-env/setup-kind.sh:137]: Yes might be possible, but I don't know how either. I think this is good enough for a local test env

**Review by glrf — APPROVED** [review]

**Pull request merged**